src/chfn.c

```diff
@@ -280,7 +276,7 @@ static void process_flags (int argc, char **argv)
 		{NULL, 0, NULL, '\0'}
 	};
 
-	/* 
+	/*
 	 * The remaining arguments will be processed one by one and executed
```

Comment:
Usually you don't make unrelated whitespace fixes in the same commit as a real change, i.e. if you do them at all, do them in a separate commit.

Reply:
dropped